package.json

```diff
@@ -72,6 +72,7 @@
         "react-native-linear-gradient": "^2.5.6",
         "react-native-safe-area-context": "^3.1.9",
         "react-native-safe-area-view": "^1.1.1",
-        "react-native-svg": "^12.1.0"
+        "react-native-svg": "^12.1.0",
+        "ripe-commons": "^0.4.0"

```

Comment:
`npm run upgrade`, so you end up with sorted entries.

Reply:
It's sorted

Reply:
🤦 my bad